repos/fairsoft/packages/pythia8/package.py

```diff
@@ -49,3 +43,7 @@ def configure_args(self):
     def setup_environment(self, spack_env, run_env):
         run_env.set('PYTHIA8DATA', '%s/share/Pythia8/xmldoc' % prefix )
         run_env.set('PYTHIA8', '%s' % prefix )
+
+    def url_for_version(self, version):
+        url = "https://pythia.org/download/pythia{0}/pythia{1}.tgz"
+        return url.format(str(version)[:2], version)
```

Comment:
Usually, spack's url-version-magic can handle two entries of the version code, even with subgroups and things.
Naive question: Did you try to put an `url` keyword and it didn't work out?
Hmm. Strange.

Reply:
After having seen this in legacy again, I realized, that for spack this is only one single version part. Not like `83.03`. So it can't see `83` as a part of `8303`.

Resolving.